content/docs/instrumenting/writing_exporters.md

```diff
@@ -229,256 +255,277 @@ metrics, stop.
 
-The first is some label it’d be useful to have on the metrics that are about,
-such as the version number of the software. Use the approach described at
-[https://www.robustperception.io/how-to-have-labels-for-machine-roles/](http://www.robustperception.io/how-to-have-labels-for-machine-roles/)
-instead.
+The first is for some label it would be useful to have on the metrics
+such as the version number of the software. Instead, use the approach
+described at
+[https://www.robustperception.io/how-to-have-labels-for-machine-roles/](http://www.robustperception.io/how-to-have-labels-for-machine-roles/).
 
-The other case are what are really target labels. These are things like region,
-cluster names, and so on, that come from your infrastructure setup rather than
-the application itself. It’s not for an application to say where it fits in
-your label taxonomy, that’s for the person running the Prometheus server to
-configure and different people monitoring the same application may give it
-different names.
+The second case is when a label is really a target label. These are
+things like region, cluster names, and so on, that come from your
+infrastructure setup rather than the application itself. It’s not for an
+application to say where it fits in your label taxonomy, that’s for the
+person running the Prometheus server to configure and different people
+monitoring the same application may give it different names.
 
-Accordingly these labels belong up in the scrape configs of Prometheus via
-whatever service discovery you’re using. It’s okay to apply the concept of
-machine roles here as well, as it’s likely useful information for at least some
-of the people scraping it.
+Accordingly, these labels belong up in the scrape configs of Prometheus
+via whatever service discovery you’re using. It’s okay to apply the
+concept of machine roles here as well, as it’s likely useful information
+for at least some people scraping it.
 
 ### Types
 
-You should try to match up the types of your metrics to Prometheus types. This
-usually means counters and gauges. The `_count` and `_sum` of summaries are
-also relatively common, and on occasion you’ll see quantiles. Histograms are
-rare, if you come across one remember that the exposition format exposes
-cumulative values.
+You should try to match up the types of your metrics to Prometheus
+types. This usually means counters and gauges. The `_count` and `_sum`
+of summaries are also relatively common, and on occasion you’ll see
+quantiles. Histograms are rare, if you come across one remember that the
+exposition format exposes cumulative values.
 
-Often it won’t be obvious what the type of a metric is (especially if you’re
-automatically processing a set of metrics), use `UNTYPED` in that case. In
-general `UNTYPED` is a safe default.
+Often it won’t be obvious what the type of metric is, especially if
+you’re automatically processing a set of metrics, use `UNTYPED` in that
```

Comment:
I think end the sentence at "metrics" might be a little clearer.

Reply:
Yeah. It's a bit repetitive.